<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVMTrain</class>
 <widget class="QWidget" name="SVMTrain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SVM training</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1,0,0,0,0">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="featureComboBox">
       <item>
        <property name="text">
         <string>Waveform length</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Feature selection:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Kernel selection:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="kernelComboBox">
       <item>
        <property name="text">
         <string>rbf</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>linear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>poly</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sigmoid</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>C selection:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="cTextField"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="progressLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Train accuracy:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="trainAcc">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="testButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Test it!</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
